MyStore/src/GeneratorData.cpp

```diff
@@ -9,9 +9,11 @@ InfoProduct GeneratorData::GeneratedProduct()
 
 	int index_color = rand() * ((max_index_colors - 0 + 0.999) / RAND_MAX) + 0;
 	int index_name = rand() * ((max_index_names - 0 + 0.999) / RAND_MAX) + 0;
+	srand(time(NULL));
 
 	data.m_product_color = m_colors[index_color];
 	data.m_product_name = m_names[index_name];
+	data.m_price = (rand() + rand()) / 130 * 2;

```

Comment:
We must be able to control minimum and maximum price of the product. Please add min and max price parameters and retrieve here random price in this range.

Reply:
Got it. I've already pushed changes.